src/mon/PGMonitor.cc

```diff
@@ -1471,8 +1474,19 @@ void PGMonitor::dump_pool_stats(stringstream &ss, Formatter *f, bool verbose)
     } else {
       tbl << pool_name
           << pool_id;
-      if (verbose)
+      if (verbose) {
 	tbl << "-";
+
+       if (pool->quota_max_objects == 0)
+         tbl << "N/A";
+       else
+         tbl << si_t(pool->quota_max_objects);
+         
+       if (pool->quota_max_bytes == 0)
+         tbl << "N/A";
+       else
+         tbl << si_t(pool->quota_max_bytes);
+      }
     }
     dump_object_stat_sum(tbl, f, stat.stats.sum, avail, verbose);
```

Comment:
Since we are modifying the table output, it makes sense to add these to the json output as well?